INSTALL

```diff
@@ -0,0 +1,368 @@
+Installation Instructions
```

Comment:
This file is automatically generated by `autogen.sh` and should not be comitted to the repository.

Reply:
done